lib/models/favorite.js

```diff
@@ -0,0 +1,12 @@
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../knexfile')[environment]
+const database = require('knex')(configuration);
+
+const all = () => database('favorites').select()
+const find = (id) => database('favorites').where('id', id).select()
+const create = (newFavorite) => database('favorites').insert(newFavorite, ['id', 'name', 'artist_name', 'genre', 'rating'])
```

Comment:
Nice work on figuring out what we needed for this to return precisely what we wanted.